### PR TITLE
New files

### PR DESCRIPTION
**Pull request by msft-poulomi**

### CODE REVIEW

File: api-reference/beta/api/authenticationmethodspolicy.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+|:---|:---|:---|
+|description|String|**TODO: Add Description**|
+|displayName|String|**TODO: Add Description**|
+|id|String|**TODO: Add Description** Inherited from [entity](../resources/entity.md).|
+|lastModifiedDateTime|DateTimeOffset|**TODO: Add Description**|
+|policyMigrationState|authenticationMethodsPolicyMigrationState|**TODO: Add Description**.The possible values are: `preMigration`, `migrationInProgress`, `migrationComplete`, `unknownFutureValue`.|
+|policyVersion|String|**TODO: Add Description**|
+|reconfirmationInDays|Int32|**TODO: Add Description**|
+|registrationEnforcement|[registrationEnforcement](../resources/registrationenforcement.md)|**TODO: Add Description**|
+|systemCredentialPreferences|[systemCredentialPreferences](../resources/systemcredentialpreferences.md)|"Credential preference for all user authentications is calculated at runtime and prompted to the user. This overrides the user-selected authentication method. Note that presently, it will be enforced only for MFA. Going forward, in future releases, this will be enforced for all authentications."|
```
Comment (jpettere) [inline api-reference/beta/api/authenticationmethodspolicy.md:40]: This file already exists (link below). You should find the existing one and add systemCredentialPreferences to it. 

https://learn.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-beta

File: api-reference/beta/api/systemcredentialpreferences.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: "authenticationMethodsPolicy resource type"
```
Comment (jpettere) [inline api-reference/beta/api/systemcredentialpreferences.md:2]: Update the resource type.

File: api-reference/beta/api/systemcredentialpreferences.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+doc_type: resourcePageType
+---
+
+# systemCredentialPreferences resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+**TODO: Add Description**
```
Comment (jpettere) [inline api-reference/beta/api/systemcredentialpreferences.md:16]: Make sure to do all of the TODO's.

File: api-reference/beta/api/systemcredentialpreferences.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|excludeTarget|[excludeTarget](../resources/excludetarget.md) collection|Users and groups included in the system preferred authentication method experience|
+|includeTarget|[includeTarget](../resources/includetarget.md) collection|Users and groups excluded from the system preferred authentication method experience|
+|state|advancedConfigState|Enable or disable the feature. Possible values are: `default`, `enabled`, `disabled`, `unknownFutureValue`. The `default` value is used when the configuration hasn't been explicitly set and uses the default behavior of Azure AD for the setting. The default value is `disabled`.|
```
Comment (jpettere) [inline api-reference/beta/api/systemcredentialpreferences.md:23]: Just double checking but the MS default value is "disabled" to start?

**Review by jpettere — CHANGES_REQUESTED** [review]
Added comments. I also think you're missing documenting the API call that can be made on an individual user to get their signInPreferences so let's make sure to add that. 

https://graph.microsoft.com/beta/users/{user UPN or ID}/authentication/signInPreferences

The /authentication/signInPreferences is not changed in this feature and is hence not available in doc stubs. This was confirmed by the engineer.

File: api-reference/beta/api/systemcredentialpreferences.md (hunk 2 above)
Comment (msft-poulomi) [reply api-reference/beta/api/systemcredentialpreferences.md:2]: Updated

File: api-reference/beta/api/systemcredentialpreferences.md (hunk 4 above)
Comment (msft-poulomi) [reply api-reference/beta/api/systemcredentialpreferences.md:23]: Yes

File: api-reference/beta/api/systemcredentialpreferences.md (hunk 3 above)
Comment (msft-poulomi) [reply api-reference/beta/api/systemcredentialpreferences.md:16]: Updated

File: api-reference/beta/resources/authenticationmethodspolicy.md [hunk 5]
```diff
@@ -22,8 +22,11 @@ Defines authentication methods and the users that are allowed to use them to sig
 |[Update authenticationMethodsPolicy](../api/authenticationmethodspolicy-update.md)|[authenticationMethodsPolicy](../resources/authenticationmethodspolicy.md)|Update the properties of an [authenticationMethodsPolicy](../resources/authenticationmethodspolicy.md) object.|
 
 ## Properties
-|Property|Type|Description|
+|Property|Type|Description|Required|ReadOnly
```
Comment (jpettere) [inline api-reference/beta/resources/authenticationmethodspolicy.md:25]: If I look at the rich diff the properties table doesn't render. You should remove the required and read-only column headers.

File: api-reference/beta/resources/authenticationmethodspolicy.md [hunk 6]
```diff
@@ -22,8 +22,11 @@ Defines authentication methods and the users that are allowed to use them to sig
...
 |:---|:---|:---|
+|state|microsoft.graph.advancedConfigState|The configuration state of the system preferred authentication method experience.|Yes|No
```
Comment (jpettere) [inline api-reference/beta/resources/authenticationmethodspolicy.md:27]: these properties (state, includeTargets and excludeTargets) should be unchanged. They are properties of authenticationmethodspolicy, not system preferred auth.

File: api-reference/beta/resources/authenticationmethodspolicy.md [hunk 7]
```diff
@@ -32,7 +35,8 @@ Defines authentication methods and the users that are allowed to use them to sig
...
 |reconfirmationInDays|Int32|Days before the user will be asked to reconfirm their method. |
 |registrationEnforcement|[registrationEnforcement](../resources/registrationenforcement.md)|Enforce registration at sign-in time. This property can be used to remind users to set up targeted authentication methods.|
-
+|systemCredentialPreferences|[systemCredentialPreferences](../resources/systemcredentialpreferences.md)|"Credential preference for all user authentications is calculated at runtime and prompted to the user. This overrides the user-selected authentication method. Note that presently, it will be enforced only for MFA. Going forward, in future releases, this will be enforced for all authentications."|
```
Comment (jpettere) [inline api-reference/beta/resources/authenticationmethodspolicy.md:38]: This seems to be missing part of the definition and remove the quotes.

File: api-reference/beta/resources/authenticationmethodspolicy.md [hunk 8]
```diff
@@ -58,6 +62,9 @@ The following is a JSON representation of the resource.
   "policyVersion": "String",
   "registrationEnforcement": {
     "@odata.type": "microsoft.graph.registrationEnforcement"
-  } 
+  } ,
+    "systemCredentialPreferences": {
```
Comment (jpettere) [inline api-reference/beta/resources/authenticationmethodspolicy.md:66]: indent this back 1 tab.

**Review by jpettere — CHANGES_REQUESTED** [review]
I think you're still missing documenting the API for:

https://graph.microsoft.com/beta/users/{UPN or ID}/authentication/signInPreferences

**Comment by Jarbas-MSFT** [comment]
Hi everyone, closing this PR because this content already exists in PR #20260.

**Pull request closed without merging**